index.html

```diff
@@ -53,5 +53,257 @@ <h2 class="section-header-subtitle">I’m a Software Developer</h2>
         </ul>
       </section>
     </section>
+
+<section class="whole-middle-section">
+    <section class="recent-work">
+        <h1 class="recent-work-content">
+            My Recent Works
+        </h1>
+
+    </section>
+
+    <section class="section-bg">
+        <div class="bg">
+        </div>
+    </section>
+
+    <section class="multi-post">
+        <h1 class="multipost-content">
+            Multi-Post Stories
+        </h1>
```

Comment:
- [ ] Kindly change the h1 to a lower hx tag because following semantic HTML on one page it should be only one h1 and it should be in the beginning. Keep rocking💪

Reply:
Okay, done!